cansniffer.c

```diff
@@ -593,11 +649,12 @@ int handle_timeo(int fd, long currcms){
 
 };
 
-void print_snifline(int id){
+void print_snifline(int position){
 
-	long diffsec  = sniftab[id].currstamp.tv_sec  - sniftab[id].laststamp.tv_sec;
-	long diffusec = sniftab[id].currstamp.tv_usec - sniftab[id].laststamp.tv_usec;
-	int dlc_diff  = sniftab[id].last.can_dlc - sniftab[id].current.can_dlc;
+	long diffsec  = sniftab[position].currstamp.tv_sec  - sniftab[position].laststamp.tv_sec;
```

Comment:
please leave only one space before and after the `-`. I know you didn't add this... :)